### DISCUSSION TITLE
Package installation not succesful

### DISCUSSION THREAD

**Discussion by Surfix** [giswqs/geodemo#6] [Q&A] [answered]
Hi Prof,

I am following your Geographic Software design tutorial playlist. I am getting an error when I tried to install my package using `pip install -e .`  I guess it has to do with Fiona or GDAL but I don't know what exactly is the problem. Kindly suggest what to do sir. Thanks 
![image](https://user-images.githubusercontent.com/51313944/112705346-444ee280-8ed9-11eb-948b-dd8ec2de0732.png)

**Comment by giswqs** [comment, marked as answer]
The `geodemo` package does not use fiona, gdal, or geopandas dependencies. Not sure why you would encounter this error. Try `conda install geodemo -c conda-forge`

```
 earthengine-api
 folium 
 ipyleaflet
 pyshp
```

**Reply by Surfix** [reply]
Thank you for the insight prof.  I am following the playlist to develop a package for working with icesat-2 data, so I put folium and icepyx in addition to the ipyleaflet dependencies in the requirements.txt file. I think one of them requires fiona,gdal and others. I have removed them and it is working fine now. Thanks
